.github/CODEOWNERS

```diff
@@ -1,4 +1,4 @@
-* @rajsite @atmgrifter00 @cameronwaterman
+* @rajsite @cameronwaterman
 
 /src/datasources/asset @CiprianAnton @kkerezsi
```

Comment:
We are missing owners for data source directories. I'll tag some folks, please confirm or mention who should own instead (based on [component ownership spreadsheet](https://emerson.sharepoint.com/:x:/r/sites/ASWSystemLink/_layouts/15/Doc.aspx?sourcedoc=%7B2D5DFFDF-EB79-43B6-8BB5-925EBE5BED08%7D&file=SystemLinkComponentOwnership.xlsx&action=default&mobileredirect=true)):

/asset-common @CiprianAnton @kkerezsi
/data-frame @maxgax who should be the data frame data source owner?
/notebook @CiprianAnton @kkerezsi 
/system @stefanc18 know who would own the systems data source?
/tag @CiprianAnton @kkerezsi 
/workspace @maxgax who should be the workspace data source owner (as it's part of user service)?